addons/binding/org.openhab.binding.avmfritz/ESH-INF/config/config.xml

```diff
@@ -42,7 +42,7 @@
 			<description>User name which has HomeAuto permissions on the given FRITZ!Box.
 			</description>
 		</parameter>
-		<parameter name="password" type="text" required="false" groupName="authentication">
+		<parameter name="password" type="text" required="true" groupName="authentication">
```

Comment:
Afair, the password on a FritzBox has been optional so far. Has this changed?

Reply:
No, not really. But there are different places in the code (see [BoxHandler.java#L104](https://github.com/cweitkamp/openhab2-addons/blob/eb7518303774845dc669d1a974ccb7b0a78a6deb/addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/BoxHandler.java#L104) and [FritzahaWebInterface.java#L92](https://github.com/cweitkamp/openhab2-addons/blob/eb7518303774845dc669d1a974ccb7b0a78a6deb/addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/FritzahaWebInterface.java#L92)) where the binding stops further progress when no password is set. So I suggest we better set it to required.